src/layouts/MainLayout.vue

```diff
@@ -64,6 +64,9 @@ export default {
 
     this.initNotifications()
     await this.fetchPeriods()
+
+    this.lightWalletLogin()
```

Comment:
please move this into the autoLogin method
it takes care of selecting the wallet for the user

here you will always login with the lightwallet and then use the other wallets
I think that in this situation we can have 2 login light wallet and Anchor for instance

Reply:
fixed